### PR TITLE
Enhancement/7685 inp tooltip

### PR DESCRIPTION
**Pull request by zutigrm**
## Summary

Addresses issue:

- #7685 

## PR Author Checklist

- [x] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 5.2 and PHP 5.6.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [x] Run the code.
- [x] Ensure the acceptance criteria are satisfied.
- [x] Reassess the implementation with the IB.
- [x] Ensure no unrelated changes are included.
- [x] Ensure CI checks pass.
- [ ] Check Storybook where applicable.
- [x] Ensure there is a QA Brief.

## Merge Reviewer Checklist

- [x] Ensure the PR has the correct target branch.
- [x] Double-check that the PR is okay to be merged.
- [x] Ensure the corresponding issue has a ZenHub release assigned.
- [x] Add a changelog message to the issue.

### CODE REVIEW

**Review by hussain-t — APPROVED** [review]
Thanks, @zutigrm. LGTM 👍 

FYI, I've added the missing prop types for `hintText`.

File: assets/js/modules/pagespeed-insights/components/common/FieldReportMetrics.js [hunk 1]
```diff
@@ -191,6 +192,15 @@ export default function FieldReportMetrics( { data, error } ) {
 						}
 						isLast
 						isUnavailable={ ! interactionToNextPaint }
+						hintText={ createInterpolateElement(
+							__(
+								'INP will replace FID in March 2024 <LearnMoreLink />',
```
Comment (nfmohit) [inline assets/js/modules/pagespeed-insights/components/common/FieldReportMetrics.js:197]: We are missing a period.

```suggestion
								'INP will replace FID in March 2024. <LearnMoreLink />',
```

File: assets/sass/components/dashboard/_googlesitekit-ReportMetric.scss [hunk 2]
```diff
@@ -77,6 +77,12 @@
 	vertical-align: bottom;
 }
 
+.googlesitekit-pagespeed-report-metric__hint-text {
+	color: $c-neutral-n-500;
+	letter-spacing: 0.25px;
```
Comment (nfmohit) [inline assets/sass/components/dashboard/_googlesitekit-ReportMetric.scss:82]: Let's use our SCSS variable for letter-spacing. Also, let's add a line-height value that is same as the referenced note.

```suggestion
	letter-spacing: $ls-s;
	line-height: $lh-body-lg;
```

File: assets/sass/components/dashboard/_googlesitekit-ReportMetric.scss [hunk 3]
```diff
@@ -77,6 +77,12 @@
...
+	margin-top: 7px;
```
Comment (nfmohit) [inline assets/sass/components/dashboard/_googlesitekit-ReportMetric.scss:83]: Let's use a relative 0.5em value as that appears to align more with the AC (half of 1em).

```suggestion
	margin-top: 7px;
```

**Review by nfmohit — CHANGES_REQUESTED** [review]
Nice work here, thank you @zutigrm.

I have left a few comments, but as they're very minor, I've gone ahead and applied them myself.

Hope that's okay, thanks!

**Review by nfmohit — APPROVED** [review]
LGTM! 👍

**Pull request merged**